Monika After Story/game/script-topics.rpy

```diff
@@ -16300,3 +16300,129 @@ label monika_gmos:
     m 3esd "Like I said before, GMOs are a complex topic. {w=0.2}If you want to learn more, make sure that your sources are reliable and that you're able to see the discussion from both sides."
     m 1eua "I think that's enough for now, thanks for listening~"
     return
+
+default persistent._mas_pm_employed = None
+default persistent._mas_pm_lookingforwork = None
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_player_employment",
+            category=['you'],
+            prompt="Employment status",
+            random=True
+        )
+    )
+
+label monika_player_employment:
+    if mas_event_mailbox.is_monika_initiated_mode():
```

Comment:
Just also an FYI to leave this out for now until we get the talk menu awareness stuff merged into upstream MAS, since
a) These aren't necessarily final
b) These don't exist in `content` right now

Reply:
I'll take these out in my next draft, no problem. :D